observability-kit-test/observability-kit-agent-it/src/test/java/com/vaadin/observability/test/AbstractViewIT.java

```diff
@@ -0,0 +1,71 @@
+package com.vaadin.observability.test;
+
+import com.vaadin.testbench.IPAddress;
+import com.vaadin.testbench.parallel.ParallelTest;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.junit.Before;
+import org.junit.BeforeClass;
+
+import java.io.File;
+
+public abstract class AbstractViewIT extends ParallelTest {
+    public static final int SERVER_PORT = Integer
+            .parseInt(System.getProperty("serverPort", "8080"));
+
+    static String hostName;
+    static boolean isHub;
+
+    @BeforeClass
+    public static void setupClass() {
+        String hubHost = System
+                .getProperty("com.vaadin.testbench.Parameters.hubHostname");
+        isHub = hubHost != null && !hubHost.isEmpty();
+        if (isHub) {
+            String driver = System.getProperty("webdriver.chrome.driver");
+            if (driver == null || !new File(driver).exists()) {
+                WebDriverManager.chromedriver().setup();
+            }
+        }
+        hostName = isHub ? IPAddress.findSiteLocalAddress() : "localhost";
+    }
+
+    @Before
```

Comment:
I suppose this JUnit 4 hook will not be executed, since it seems the module uses only Junit 5

Reply:
let replace it with @BeforeAll and @BeforeEach